docs/configuration.html

```diff
@@ -36,22 +36,22 @@
   <a id="topic-config" href="#topic-config">Topic-level configuration</a>
 
   Configurations pertinent to topics have both a server default as well an optional per-topic override. If no per-topic configuration is given the server default is used. The override can be set at topic creation time by giving one or more <code>--config</code> options. This example creates a topic named <i>my-topic</i> with a custom max message size and flush rate:
-  <pre>
+  <pre class="brush: bash;">
   <b> &gt; bin/kafka-topics.sh --zookeeper localhost:2181 --create --topic my-topic --partitions 1
```

Comment:
The classes you're using for the pre tags looks good, but it looks like there was previously html tags for formatting (In this case I see a bold tag) in the code snippets that aren't "bash" or "java" etc.